ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -38,15 +40,55 @@ def __init__(self):
 
         # TODO: Add other member variables you need below
 
+        # Current pose
+        self.pose = None
+
+        # Base waypoints
+        self.waypoints = None
+
         rospy.spin()
 
     def pose_cb(self, msg):
```

Comment:
Can you add comments regarding the algorithm you've used and steps you took?
The code doesn't look that trivial...